src/components/jobs/PayoutEmailPreview.tsx

```
@@ -0,0 +1,269 @@
+          )}
+
+          {/* Preview Content */}
+          {selectedAttachment && (
+            <Tabs defaultValue="email" className="flex-1 flex flex-col overflow-hidden">
+              <TabsList>
+                <TabsTrigger value="email">Email HTML</TabsTrigger>
+                <TabsTrigger value="data">Datos</TabsTrigger>
+              </TabsList>
+
+              <TabsContent value="email" className="flex-1 overflow-auto border rounded-md p-4">
+                <div dangerouslySetInnerHTML={{ __html: getEmailHTML() }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)